us_imputation_benchmarking/models/imputer.py

```diff
@@ -0,0 +1,88 @@
+from abc import ABC, abstractmethod
+from typing import Any, Dict, List, Optional, Set, Union
+
+import numpy as np
+import pandas as pd
+
+from us_imputation_benchmarking.utils.logging_utils import get_logger
+
+
+class Imputer(ABC):
+    """
```

Comment:
Can we add the Imputer general variables here? I don't think the initialiser uses the random seed.

predictors: List[str] | None
"""Description"""